client/components/jetpack/daily-backup-status/status-card/backup-realtime-message.tsx

```diff
@@ -0,0 +1,83 @@
+import { useTranslate } from 'i18n-calypso';
+import { FunctionComponent } from 'react';
+import { useLocalizedMoment } from 'calypso/components/localized-moment';
+
+type Props = {
+	backupDate: number; // timestamp
+	eventsCount: number;
+};
+
+export const BackupRealtimeMessage: FunctionComponent< Props > = ( {
+	backupDate,
+	eventsCount,
+} ) => {
+	const translate = useTranslate();
+	const moment = useLocalizedMoment();
+	const backupDateMoment = moment( backupDate );
+	const today = moment();
```

Comment:
Instead of comparing it with `today` we should compare it with the date of the backup selected